backend/tests/test.cpp

```diff
@@ -0,0 +1,29 @@
+#define CATCH_CONFIG_MAIN
```

Comment:
**question**: Why is this here?

Reply:
creates the "main" program to run the tests